ql/src/test/results/clientpositive/llap/materialized_view_create_rewrite_3.q.out

```diff
@@ -227,7 +227,9 @@ STAGE PLANS:
           Table: default.cmv_mat_view
 
   Stage: Stage-5
-    Materialized View Work
+    Materialized View Update
+      name: default.cmv_mat_view
+      retrievAndInclude: true
```

Comment:
Typo?

Reply:
Thanks, nice catch, fixed.

Reply:
Fixed some checkstyle issues and a failing q test.